### ISSUE TITLE
introspect:sqlite generates invalid schema for text columns

### ISSUE THREAD

**Issue by ninxadev** [drizzle-team/drizzle-kit-mirror#116]
Some of my text columns were generated with default(NULL) in the schema.ts file. This throws a typescript error (bc NULL isn't the same as null ) but it should probably not be generating any default value since text columns are NULL by default.

This happens for existing databases with column that has some NULL values mixed with some TEXT values.

*Note: The outputted migration files are fine, this is only a schema.ts issue*

```typescript
// drizzle.config.ts
export default {
introspect: {
        casing: 'camel',
    },
    driver: 'better-sqlite',
    out: './drizzle',
    dbCredentials: {
        url: './drizzle/chat-test.db',
    },
} satisfies Config
```

Schema generated by ```drizzle-kit introspect:sqlite```
```typescript
// drizzle/schema.ts
export const handle = sqliteTable('handle', {
    ...
    personCentricId: text('person_centric_id').default(NULL),
})
```

Error:
```zsh
drizzle/schema.ts:73:56 - error TS2304: Cannot find name 'NULL'.

73     personCentricId: text('person_centric_id').default(NULL),
```

I found this in the source that might be causing it:

```typescript
// src/sqlite-introspect.ts
mapColumnDefault2 = (defaultValue) => {
      if (typeof defaultValue === "string" && defaultValue.startsWith("(") && defaultValue.endsWith(")")) {
        return `sql\`${defaultValue}\``;
      }
      return defaultValue;
    };
```

This fixes the error and maintains the migration files, but is probably not the ideal solution.
```typescript
export const NULL = sql`NULL`
```

Let me know if you need anymore details, I have reproduced it a few times.

Hope it helps!

**Comment by AndriiSherman** [comment]
Yeah, thanks. Let me fix it now. WIll mark you as a contributor for this fix

**Comment by AndriiSherman** [comment]
Fixed in `drizzle-kit@0.19.5`, thanks!
